eShopModernizedMVCSolution/src/eShopModernizedMVC/Modules/ApplicationModule.cs

```diff
@@ -52,6 +54,19 @@ protected override void Load(ContainerBuilder builder)
 
             builder.RegisterType<CatalogItemHiLoGenerator>()
                 .SingleInstance();
+
+            if (this.useManagedIdentity)
+            {
+                builder.RegisterType<ManagedIdentitySqlAccessTokenProvider>()
```

Comment:
@twsouthwick I'm not sure how I didn't catch this before, but it looks like these types are missing. 
The `ManagedIdentitySqlAccessTokenProvider`, `ISqlAccessTokenProvider`, and `AppSettingsSqlAccessTokenProvider`.

Did they just get missed? If so, would you mind opening another PR to add them?